varnish-mixin/dashboards/varnish-overview.libsonnet

```diff
@@ -9,6 +9,10 @@ local dashboardUid = 'varnish-overview';
 local promDatasourceName = 'prometheus_datasource';
 local lokiDatasourceName = 'loki_datasource';
 
+local getMatcher(cfg) = '%(varnishSelector)s, instance=~"$instance"' % cfg;
+local frontendLogFilter = 'filename=~"/var/log/varnish/varnishncsa-frontend.*.log|/opt/varnish/log/varnishncsa-frontend.*.log"';
+local backendLogFilter = 'filename=~"/var/log/varnish/varnishncsa-backend.*.log|/opt/varnish/log/varnishncsa-backend.*.log"';
```

Comment:
Just a thought, but do we want to call out the two potential log paths in the readme with a section for k8s setup, along with an explanation for users of the mixin to check that it matches their deployments paths?

If there are no other variations in path than these two, then I think we're good to omit that

Reply:
Where the file(s) are is completely customizable, that's my understanding. As long as the file/directory permissions are set correctly and whatnot, so I think we need to call out these log paths as something to be matched. 👍